txpipe/source_selector.py

```diff
@@ -28,8 +58,8 @@ class TXSourceSelector(PipelineStage):
     TODO: add option to use lensfit catalogs, which
     would be much much simpler.
     """
+    name = "TXSourceSelector"
```

Comment:
is this still a viable stage? do we actually need to name it?

Reply:
I've added a note on this now - there is a check later to make sure that if anyone is still using the old name then it will complain and tell them how to change it.